runelite-client/src/main/java/net/runelite/client/plugins/microbot/zerozero/tormenteddemons/TormentedDemonPlugin.java

```diff
@@ -0,0 +1,138 @@
+package net.runelite.client.plugins.microbot.zerozero.tormenteddemons;
+
+import com.google.inject.Provides;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.api.GraphicsObject;
+import net.runelite.api.InventoryID;
+import net.runelite.api.Item;
+import net.runelite.api.ItemComposition;
+import net.runelite.api.events.GraphicsObjectCreated;
+import net.runelite.client.config.ConfigManager;
+import net.runelite.client.eventbus.Subscribe;
+import net.runelite.client.events.ConfigChanged;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.client.plugins.microbot.Microbot;
+
+import java.awt.datatransfer.StringSelection;
+import java.util.StringJoiner;
+import java.util.concurrent.Executors;
+import net.runelite.client.plugins.microbot.util.tile.Rs2Tile;
+import net.runelite.client.ui.overlay.OverlayManager;
+
+import javax.inject.Inject;
+import java.awt.*;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+@PluginDescriptor(
+        name = PluginDescriptor.zerozero + "Tormented Demons",
+        description = "Automates restocking, prayer flicking, and gear switching during Tormented Demon",
+        tags = {"tormented", "flicker", "weapon", "switch"},
```

Comment:
Can you add "microbot" as a tag? it's easier to filter if people are searching for microbot scripts